dev-util/pkgconf/pkgconf-1.0.1.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="git://github.com/pkgconf/pkgconf.git"
```

Comment:
Please add a https:// source as well, and as the first (preferred) one. This gives people the little bit of extra security via HTTPS certificate verification.